zephyr/west.yml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+manifest:
+  self:
+    path: wpa_supplicant_zephyr
+
+  remotes:
+    - name: zephyrproject-rtos
+      url-base: https://github.com/zephyrproject-rtos
+
+  projects:
+    - name: zephyr
+      remote: zephyrproject-rtos
```

Comment:
This is a bit confusing, why is the module pulling the upstream Zephyr? It should rather work with `sdk-zephyr` pulled by `sdk-nrf` manifest file.

Reply:
Removing leftover files.